src/components/picker/PickerItem.js

```diff
@@ -23,20 +22,19 @@ class PickerItem extends BaseComponent {
 
   static propTypes = {
     /**
-     * [DEPRECATED - please include the label in the value prop] The item label
```

Comment:
Please remove the TODO at the top of the file to avoid confusion.

Reply:
Removed!